docker/kibana/8.11.1/Saved Objects/Apache Flagon Business Analytics Dashboard.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
The saved object format was changed, and this are no longer compatible with 8.11. The files in this directory should be .ndjson instead of .json

Reply:
After your changes I still can't import these files as saved objects.

`The file could not be processed due to error: "Bad Request: Unexpected end of JSON input"`